src/bundle/Resources/views/link_manager/view.html.twig

```diff
@@ -0,0 +1,125 @@
+{% extends 'EzPlatformAdminUiBundle::layout.html.twig' %}
+
+{% trans_default_domain "linkmanager" %}
+
+{%- block breadcrumbs -%}
+    {% include '@EzPlatformAdminUi/parts/breadcrumbs.html.twig' with { items: [
+        { value: 'url.list'|trans, url: path('ezplatform.link_manager.list') },
+        { value: 'url.detail'|trans|desc('Link detail')}
+    ]} %}
+{%- endblock -%}
+
+{%- block pageTitle -%}
+    {% include '@EzPlatformAdminUi/parts/page_title.html.twig' with {
+        title: url.url|truncate(60),
+        iconName: 'link'
+    } %}
+{%- endblock -%}
+
+{% block bodyClass %}ez-info-view{% endblock %}
```

Comment:
This class is for `System Information`, please use `ez-link-manager-view` or something similar.